uber_sante/controllers/__init__.py

```diff
@@ -1,6 +1,8 @@
 from flask import Blueprint
+
 controllers = Blueprint('controllers', __name__)
 
-from .availability_controller import *
-from .booking_controller import *
-from .patient_controller import *
+
```

Comment:
You cannot remove these 3 lines or else no endpoint is accessible from the /controllers folder

Reply:
My bad, fixing